### DISCUSSION TITLE
休息 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#420] [General]
# 休息 - Miao Yu | 于淼

https://yufree.cn/cn/2023/05/29/have-a-rest/

**Comment by NetZissou** [comment]
> 因为R跟python都是高级语言，所以应用上互通的地方又很多，有点类似看日文的感觉
实在是太形象了哈哈！不知道于淼今年会不会参加Posit Conf？

**Reply by yufree** [reply]
应该不会，我都是会开完了去浏览下幻灯片云一下，外出开会比上班累多了
